.eslintrc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   root: true,
-  extends: '@react-native-community',
+  extends: '@react-native',
```

Comment:
Veerji, it would be good if we use airbnb eslint styleguide.

Reply:
Hnji, that's a known issue, https://github.com/KhalisFoundation/learn-larivaar/issues/31